conf.py

```diff
@@ -161,16 +161,21 @@
         ),
         (
           (
+            ("/mission-statement/", "Our mission statement", ""),
             ("/how-ansible-works/", "How Ansible works", ""),
             ("/ecosystem/", "Ansible ecosystem", ""),
+            ("/awx/", "AWX", ""),
+            ("/galaxy/", "Galaxy", ""),
+            ("/faq/", "FAQs", ""),
             ("/ansible-community-training/", "Ansible community training", ""),
-            ("/mission-statement/", "Our mission statement", ""),
+            
             ("/contact-us/", "Contact us", ""),
           ),
           "Resources", ""
         ),
-        ("https://forum.ansible.com/", "Ansible community forum", ""),
+        ("https://forum.ansible.com/", "Forum", ""),
```

Comment:
We explicitly chose "Ansible community forum" to make it clear that it is not somewhere RH customers should go to find support etc.

Reply:
This was an intentional and suggested change I prefer we uphold.  The label change will not promote or prevent customers from attempting to find support at the forum.  The link label as it is was commented at appearing unnecessarily redundant.